setup.py

```diff
@@ -213,7 +213,7 @@ def run(self):
     'pysha3==0.3',
     'pytest==2.9.2',
     'pytest-cov==2.2.1',
-    'python-bitcoinlib==0.7.0',
+    'python-bitcoinlib', #==0.7.0
```

Comment:
Ahh - good.  We'll still need to doublecheck if a minor version update like 0.7.1 is released.  But that should be only a bugfix release anyway.